### PR TITLE
feat: added font jura

### PR DESCRIPTION
**Pull request by martinvibes**
@b-j-roberts kindly review  : )

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@material-tailwind/react": "^2.1.9",
     "@monaco-editor/react": "^4.6.0",
+    "@next/font": "^14.2.7",
```
Comment (b-j-roberts) [inline frontend/package.json:14]: I think `next/font` is already imported from `next`, do we need to include this import? If not we can remove it and delete changes in `frontend/package-lock.json`

File: frontend/src/components/script-editor.tsx [hunk 2]
```diff
@@ -71,14 +74,18 @@ export default function ScriptEditor() {
         )}
       >
         <Editor
-          beforeMount={setEditorTheme}
+          beforeMount={(monaco) => {
+            setEditorTheme(monaco);
+            monaco.editor.remeasureFonts();
```
Comment (b-j-roberts) [inline frontend/src/components/script-editor.tsx:79]: Can we move `monaco.editor.remeasureFonts()` into the `setEditorTheme` function?

**Review by b-j-roberts — CHANGES_REQUESTED** [review]

**Comment by martinvibes** [comment]
@b-j-roberts updated requested changes kindly review :)

**Review by b-j-roberts — APPROVED** [review]
Thank you, nice work!

**Pull request merged**